buf.gen.yaml

```diff
@@ -18,6 +18,3 @@ plugins:
     out: .
     opt:
       - allow_repeated_fields_in_body=true
-      - atlas_patch=true
```

Comment:
we'll need this option on to verify the PR works while running buf

Reply:
I am verifying it with unit tests [here](https://github.com/infobloxopen/grpc-gateway/pull/29/files#diff-63b42a18417e8e7f785294160cadfbde2399feb95d96d940c1ebb39759bbacbaR153). For simplicity and further merges with the original repo I would leave examples untouched or it might be a separate task.

Reply:
And [here](https://github.com/infobloxopen/grpc-gateway/pull/29/files#diff-d3e80c3693cab5b43b1fafe81c80fcb1093f1564144efa7a25586c09355e61faR1) we have wanted swagger json doc to compare generated result against. So this is the ideal place to keep a source of truth. Examples are too complicated to spot and document our modification changes.